mcp update (#446)
OctoMind-dev/mintlify-docs: changelog/changelog.mdx

icon: "sparkles"
---

## April 2025

### 2025-04-02

> - new octomind mcp server. see [mcp](/install-octomind-mcp) for details. You can easily integrate it into your agent 

Reviewer: ```suggestion
- `Octomind MCP` server: See [mcp](/install-octomind-mcp) for details. You can easily integrate Octomind with your AI tooling
```
